pkg/proxy/register_test.go

```diff
@@ -117,3 +117,76 @@ func createRegister(r router.Router) *Register {
 
 	return register
 }
+
+func TestParametersReplacement(t *testing.T) {
+	log.SetOutput(ioutil.Discard)
+
+	def := &Definition{
+		ListenPath: "/api/recipes/{id:[\\da-f]{24}}",
+		Upstreams: &Upstreams{
+			Balancing: "roundrobin",
+			Targets:   []*Target{{Target: "http://localhost:9089/recipes/{id}"}},
+		},
+		Methods:    []string{"ALL"},
+		StripPath:  false,
+		AppendPath: false,
+	}
+	route := NewRoute(def)
+
+	chiRouter := router.NewChiRouter()
+
+	statsClient, _ := stats.NewClient("noop://", "")
```

Comment:
Always catch errors, for testing I'd recommend `require.NoError(t, err)`, just remember to import `github.com/stretchr/testify/require`

Reply:
@rafaeljesus not necessary in this case